src/app/organizations/organizations/organizations.component.ts

```diff
@@ -13,9 +13,12 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
-import { Component, OnInit } from '@angular/core';
+import { AfterViewInit, ChangeDetectorRef, Component, OnInit, ViewChild } from '@angular/core';
```

Comment:
So I'm referring to this link: [paginator but not with table](https://stackblitz.com/edit/mat-paginator-with-cards?file=app%2Fapp.component.ts) and also the `search-entry-table.ts`